src/components/Modal/Modal.tsx

```diff
@@ -70,13 +72,13 @@ const getUniqueID = createUniqueIDFactory('modal-header');
 
 class Modal extends React.Component<CombinedProps, State> {
   static Section = Section;
-  focusReturnPointNode: HTMLElement;
```

Comment:
@danrosenthal I'm removing this, along with [this line](https://github.com/Shopify/polaris-react/pull/2206/files#diff-eaf22cccaf0b1e13dc39209235b7229fL208-L209) that should never run.

Reply:
I _think_ this should be fine, but I don't have a lot of context on how this code was used.

@tmlayton do you understand how this was used, and if it is now superfluous with these changes?

Reply:
Yes, this is safe to remove. The previous versions of modal that used the EASDK and App Bridge had to cache the `activeElement` because the modal was rendered in the parent window (different DOM). Then when the modal closed focused was restored to that cached `activeElement`.